web/src/app/schedules/ScheduleShiftList.js

```diff
@@ -107,6 +108,86 @@ function ScheduleShiftList({ scheduleID }) {
     },
   })
 
+  function getShiftDetails(s, day) {
+    const tzAbbr = DateTime.local({ zone }).toFormat('ZZZZ')
+    const localTzAbbr = DateTime.local({ zone: 'local' }).toFormat('ZZZZ')
+
+    const locale = {
+      hour: 'numeric',
+      minute: 'numeric',
+    }
+
+    let shiftDetails = ''
+    const startTime = s.start.toLocaleString(locale)
+    const endTime = s.end.toLocaleString(locale)
+    const localStartTime = DateTime.fromISO(s.start, {
+      zone: 'local',
+    }).toLocaleString(locale)
+    const localEndTime = DateTime.fromISO(s.end, {
+      zone: 'local',
+    }).toLocaleString(locale)
+
+    // shift (s.interval) spans all day
+    if (s.interval.engulfs(day)) {
+      return 'All day'
+    }
+    if (day.engulfs(s.interval)) {
+      // shift is inside the day
+      shiftDetails = `From ${startTime} to ${endTime} ${tzAbbr}`
+      if (isLocalZone) {
+        return shiftDetails
+      }
+      return (
+        <Tooltip
+          title={`From ${localStartTime} to ${localEndTime} ${localTzAbbr}`}
+          placement='bottom-start'
+          PopperProps={{
+            'aria-label': 'local-timezone-tooltip',
```

Comment:
aria-label should be left out for these -- the screen reader should read the contents of the tooltip